google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -96,6 +96,7 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   private final Boolean retentionPolicyIsLocked;
   private final Long retentionPeriod;
   private final IamConfiguration iamConfiguration;
+  private final String locationType;
```

Comment:
Can we make this field Enum?

So the logic is that the locationType would be a readonly field in Bucket. At any point in time, it would only be one of these three values if not then we must throw an exception.
`"multi-region", "region", "dual-region"`

An enum would be better suited for this.

Reply:
no because there is location field which can be also one of few values and even in that also no exception thrown in that so i think String datatype is fine.